src/praisonai/praisonai/train.py

```
@@ -1,590 +1,25 @@
-    args = parser.parse_args()
+from praisonai.train.llm.trainer import (  # noqa: F401
+    TrainModel,
+    formatting_prompts_func,
+    tokenize_function,
+    main,
+)
 
-    if args.command == "train":
-        trainer_obj = TrainModel(config_path=args.config)
-        trainer_obj.run()
+__all__ = ["TrainModel", "formatting_prompts_func", "tokenize_function", "main"]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)